### DISCUSSION TITLE
more coins setup or startup

### DISCUSSION THREAD

**Discussion by miningpirate** [coinfoundry/miningcore#901] [Q&A]
pls help 

i want more coins on me site i have now one i dont no how i can setup more coins in one config

**Comment by calvintam236** [comment]
https://github.com/coinfoundry/miningcore/wiki/Configuration

**Reply by seperatis** [reply]
Yes, i think you are correct. I can help with this. I will make example configs for multi-coin pool.

**Reply by calvintam236** [reply]
I just added comments to the config wiki. I think it is not commented correctly how to do multi coins

**Reply by GoZippy** [reply]
someone please share multi-coin config and explain CREATE TABLE shares_mypool1 PARTITION OF shares FOR VALUES IN ('mypool1'); for each so we can be sure to correct db tables for each pool... Thanks in advance... would like to help with documentation but still struggling getting this to work on my own

**Reply by oliverw** [reply]
> someone please share multi-coin config and explain CREATE TABLE shares_mypool1 PARTITION OF shares FOR VALUES IN ('mypool1'); for each so we can be sure to correct db tables for each pool... Thanks in advance... would like to help with documentation but still struggling getting this to work on my own

What do you need an explanation for? Or did you mean the Postgres `explain`?

**Reply by GoZippy** [reply]
please post multicoin config file;
please post info on each pool setup to avoid confusion - each coin has its own pool, per instruction need to add table for each pool but slight confustion on what name for each table per config setup to match.

**Comment by riksarchen** [comment]
I'm also looking for this. I managed to get pool running but webui is not working with two coins. I assume some additional config is necessary or adjusted from one coin pools.

**Reply by GoZippy** [reply]
if you figure it out can you please share? I would like to do a community wiki we can all add to

**Reply by MiningCryptoLive** [reply]
> I'm also looking for this. I managed to get pool running but webui is not working with two coins. I assume some additional config is necessary or adjusted from one coin pools. 

What webui are you using? If you use AlphaXprojects webui it supports multiple coins

**Reply by minershaven** [reply]
There are a lot of single coin UI's out there that take a bit of modifications to work with multiple coins.  

https://github.com/minernl/Miningcore.WebUI
and
https://github.com/AlphaX-Projects/alphaxcore-WebUI 
 
Those are both good UI's that support multiple coins.
